modules/ossm-accessing-bookinfo-application-using-gateway-api.adoc

```diff
@@ -3,10 +3,14 @@
 = Accessing the Bookinfo application by using Gateway API
 :context: ossm-accessing-bookinfo-application-using-gateway-API
 
-The Gateway API deploys a gateway by creating a `Gateway` resource. In {ocp-product-title} 4.15 and later versions, the Gateway API CRDs are not available by default
-and must be enabled in order to be used. 
+The {k8s} Gateway API deploys a gateway by creating a `Gateway` resource. In {ocp-product-title} 4.15 and later versions. If you want your cluster to use the Gateway API CRDs, you must enable the CRDs because they are disable by default.  
```

Comment:
```suggestion
The {k8s} Gateway API deploys a gateway by creating a `Gateway` resource. In {ocp-product-title} 4.15 and later versions. If you want your cluster to use the Gateway API CRDs, you must enable the CRDs because they are disabled by default.  
```

Reply:
Ugh! Done, thank you.